cloudify_cli/tests/commands/test_instances.py

```diff
@@ -0,0 +1,61 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+"""
+Tests all commands that start with 'cfy instances'
+"""
+
+from uuid import uuid4
+
+from mock import MagicMock
+from cloudify_rest_client.node_instances import NodeInstance
```

Comment:
this import should be one line below (i.e. separated with a blank line from "mock", and not separated from the cloudify_cli imports)